components/shared/BetSlipCard/index.tsx

```diff
@@ -0,0 +1,100 @@
+'use client';
+
+import BitTorentCoinIcon from '@/components/icons/BitTorentCoinIcon';
+import LogoWithText from '../LogoWithText';
+import ArrowDownIcon from '@/components/icons/ArrowDownIcon';
+import XIcon from '@/components/icons/XIcon';
+import { Listbox, Transition } from '@headlessui/react';
+import { useState, Fragment } from 'react';
+
+const bids = [5, 10, 20, 30, 50, 100];
+
+function BetSlipCard() {
+  const [selected, setSelected] = useState<number>(bids[0]);
+
+  return (
+    <div className="w-full bg-xport-gray-alternate text-xs rounded">
+      <div className="flex justify-between items-center px-3 py-2 ">
+        <LogoWithText className="h-7 w-16" />
+        <span>
+          <XIcon className="w-3 h-3" />
+        </span>
+      </div>
+      <div className="bg-xport-gray-primary px-3 py-2 flex flex-col gap-2">
+        <div className="flex gap-2 items-center">
+          <div className="aspect-square w-5 basis-[15%] rounded-full bg-xport-black-light animate-pulse"></div>
+          <span className="font-semibold text-xport-game-mlbb">
+            Evos Esports vs RRQ Esports
+          </span>
+        </div>
+        <div className="flex justify-between items-center font-semibold">
+          <div className="flex items-center gap-2">
+            <div className="h-5 w-5 rounded-full bg-xport-black-light animate-pulse"></div>
+            <span>Evos Esport</span>
```

Comment:
this name should be dynamic regarding what's user click on this button 
<img width="866" alt="Screen Shot 2023-09-20 at 18 02 11" src="https://github.com/galihpermana29/x-sports/assets/54941568/f2195f68-ce9c-4212-b28a-e6e3a7fcbd4e">